test/util/testUtils.js

```diff
@@ -0,0 +1,52 @@
+const mysql = require('mysql');
+const fs = require('fs');
+const path = require('path');
+
+const sqlFilePath = './dbReset.sql';
+const uploadsPath = '../../uploads';
+
+module.exports = {
+    resetDB: function () {
+        // reset database by sql script
+        fs.readFile(sqlFilePath, function (err, data) {
```

Comment:
Using just the relativ path makes it impossible to execute the script from another directory. I will figure out a solution how we can dynamically add the absolute directory of the script as a prefix.